src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,14 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val korisnik = User().apply(initUser)
+    return korisnik
 }
 
 internal fun User.address(initAddress: Address.() -> Unit): User {
-    TODO("Implement me!!!")
+    //val adresa = Address()
```

Comment:
nit: do not push the commented-out code to PRs. 

This is the solution! 👏🏻 It can also become a one-liner if you use an additional `apply` scoped function. 
Example:
```return apply { this.address = Address().apply(initAddress) }```